Kickstarter-iOS/Views/Controllers/VideoViewController.swift

```diff
@@ -88,10 +88,10 @@ public final class VideoViewController: UIViewController {
         }, completion: nil)
     }
 
-    self.viewModel.outputs.addCompletionObserver
```

Comment:
Remove `..Observer` suffix from the VM to stay consistent with the rest of the codebase